inc/cannabis-importer.php

```
@@ -0,0 +1,462 @@
+ * Handles import of cannabis products, strains, and locations
+ * with proper ACF field mapping and taxonomy assignment
+ */
+
+// Register WP-CLI commands if available
+if ( defined( 'WP_CLI' ) && WP_CLI ) {
+    WP_CLI::add_command( 'skyworld import_products', 'skyworld_cli_import_products' );
+    WP_CLI::add_command( 'skyworld import_strains', 'skyworld_cli_import_strains' );
+    WP_CLI::add_command( 'skyworld import_locations', 'skyworld_cli_import_locations' );
+}
+
+function skyworld_cli_import_products( $args, $assoc_args ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$assoc_args'. (undefined)

(UnusedFormalParameter)